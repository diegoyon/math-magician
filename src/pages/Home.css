.home {
  display: flex;
  flex-direction: column;
  gap: 1rem;
  padding: 5rem;
}

.home h2 {
  font-size: 3rem;
  margin-bottom: 2rem;
}
