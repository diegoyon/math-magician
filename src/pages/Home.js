import React from 'react';
import './Home.css';

const Home = () => (
  <div className="home">
    <h2>Welcome to our page!</h2>
    <p>
      This is a page where you can easily make some calculations
      using the calculator provided. You can access the calculator
      by clicking &quot;Calculator&quot; in the navbar.
    </p>
    <p>
      Additionally, you can read some random and interesting quotes
      about math. Go ahead and click &quot;Quote&quot; in the navbar to read
      an amazing quote!
    </p>
  </div>
);

export default Home;
