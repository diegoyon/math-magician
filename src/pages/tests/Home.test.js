import renderer from 'react-test-renderer';
import Home from '../Home.js';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});