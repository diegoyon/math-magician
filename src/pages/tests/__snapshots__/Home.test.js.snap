// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptas id consequatur quasi iure libero ab cumque vero quia sapiente inventore, ipsam voluptatem dolorum eaque! Reiciendis iure nisi corporis ipsam quidem.
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptas id consequatur quasi iure libero ab cumque vero quia sapiente inventore, ipsam voluptatem dolorum eaque! Reiciendis iure nisi corporis ipsam quidem.
  </p>
</div>
`;
